src/index.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Google LLC
+// Copyright 2020 Google LLC
```

Comment:
Restore this one?

Reply:
This file is generated by the post processor, so if I change it, the post processor will change it back.